components/card/src/css/vertical-card.css

```diff
@@ -0,0 +1,120 @@
+@import url(../../../../css/component.css);
+.jersey-container {
```

Comment:
class name should be more specific ... if in next project you change the jersey to grocery then these class names will become irrelevant

Reply:
i have created the components keeping the ecom website in mind.

Reply:
but we will use if for all projects so instead of jersey you could use card-vertical-container or card-vertical-img... Just a suggestion ... again opinions change